flask_resty/api.py

```diff
@@ -28,6 +27,9 @@ def handle_api_error(error):
     def _get_app(self, app):
         app = app or self._app
         assert app, "no application specified"
+        # store the view urls for reference. Doesn't support multiple routes
+        # mapped to same view
+        app.extensions.setdefault('resty', FlaskRestyState(self))
```

Comment:
because `app` could be None there. Also, this works if more than one app is provided during resource  registration